scalableminds/webknossos
# Hide mode toggle for 2d-only datasets

This PR hides the orthogonal/flight/oblique toggle for 2d-only datasets

### URL of deployed dev instance (used for testing):
- https://___.webknossos.xyz

### Steps to test:
- open a tracing or view of a 2d dataset
- there should be no orthogonal/flight/oblique toggle in the top menu bar

### Issues:
- fixes #4880

------
